<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MeWindow</class>
 <widget class="QMainWindow" name="MeWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>836</width>
    <height>452</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Custom menu editor</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <layout class="QVBoxLayout" name="verticalLayout_2">
    <item>
     <widget class="QSplitter" name="splitter">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <widget class="MeTree" name="treeWidget">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
         <horstretch>1</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="maximumSize">
        <size>
         <width>300</width>
         <height>16777215</height>
        </size>
       </property>
       <column>
        <property name="text">
         <string notr="true">1</string>
        </property>
       </column>
      </widget>
      <widget class="QWidget" name="details" native="true">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
         <horstretch>3</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout">
        <property name="spacing">
         <number>2</number>
        </property>
        <item>
         <spacer name="verticalSpacer_4">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label">
          <property name="text">
           <string>Label:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edName"/>
        </item>
        <item>
         <spacer name="verticalSpacer_3">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Command:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="edExec"/>
        </item>
        <item>
         <spacer name="verticalSpacer_2">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeType">
           <enum>QSizePolicy::Fixed</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>13</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Icon file:</string>
            </property>
            <property name="alignment">
             <set>Qt::AlignBottom|Qt::AlignLeading|Qt::AlignLeft</set>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QToolButton" name="tbPickIcon">
            <property name="toolTip">
             <string>Choose an icon ...</string>
            </property>
            <property name="text">
             <string>...</string>
            </property>
            <property name="iconSize">
             <size>
              <width>16</width>
              <height>16</height>
             </size>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QLineEdit" name="edIcon"/>
        </item>
        <item>
         <spacer name="verticalSpacer">
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>20</width>
            <height>40</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </widget>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QToolBar" name="toolBar">
   <property name="windowTitle">
    <string>toolBar</string>
   </property>
   <property name="iconSize">
    <size>
     <width>16</width>
     <height>16</height>
    </size>
   </property>
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNewMenu"/>
   <addaction name="actionNewItem"/>
   <addaction name="actionNewSep"/>
   <addaction name="separator"/>
   <addaction name="actionDel"/>
   <addaction name="separator"/>
   <addaction name="actionSave"/>
   <addaction name="actionApp"/>
  </widget>
  <action name="actionNewItem">
   <property name="icon">
    <iconset resource="../data/imgs.qrc">
     <normaloff>:/16/file_new</normaloff>:/16/file_new</iconset>
   </property>
   <property name="text">
    <string>New menu item</string>
   </property>
   <property name="toolTip">
    <string>New menu item</string>
   </property>
   <property name="shortcut">
    <string>N</string>
   </property>
  </action>
  <action name="actionDel">
   <property name="icon">
    <iconset resource="../data/imgs.qrc">
     <normaloff>:/16/16/delete.png</normaloff>:/16/16/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete menu item</string>
   </property>
   <property name="toolTip">
    <string>Delete menu item</string>
   </property>
   <property name="shortcut">
    <string>D</string>
   </property>
  </action>
  <action name="actionApp">
   <property name="icon">
    <iconset resource="../data/imgs.qrc">
     <normaloff>:/16/16/accept.png</normaloff>:/16/16/accept.png</iconset>
   </property>
   <property name="text">
    <string>Apply</string>
   </property>
   <property name="toolTip">
    <string>Apply</string>
   </property>
   <property name="shortcut">
    <string>A</string>
   </property>
  </action>
  <action name="actionNewMenu">
   <property name="icon">
    <iconset resource="../data/imgs.qrc">
     <normaloff>:/16/folder_new</normaloff>:/16/folder_new</iconset>
   </property>
   <property name="text">
    <string>New menu</string>
   </property>
   <property name="toolTip">
    <string>New menu</string>
   </property>
  </action>
  <action name="actionNewSep">
   <property name="icon">
    <iconset resource="../data/imgs.qrc">
     <normaloff>:/16/16/add.png</normaloff>:/16/16/add.png</iconset>
   </property>
   <property name="text">
    <string>New separator</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="../data/imgs.qrc">
     <normaloff>:/16/save</normaloff>:/16/save</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
   <property name="toolTip">
    <string>Save</string>
   </property>
  </action>
 </widget>
 <customwidgets>
  <customwidget>
   <class>MeTree</class>
   <extends>QTreeWidget</extends>
   <header>metree.h</header>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../data/imgs.qrc"/>
  <include location="../data/imgs.qrc"/>
 </resources>
 <connections/>
</ui>
